src/lib/converter.js

```diff
@@ -88,6 +108,9 @@ export const handleConversion = ({
   prefix = '',
   variableRegex,
   outputType = 'triton_placeholders',
+  langSyntax = '',
+  argsSyntax = '',
+  argSyntax = '',
```

Comment:
```suggestion
  langSyntax = 'lang',
  argsSyntax = 'args',
  argSyntax = 'arg',
```

This should fix the failing tests :)

Reply:
Added :)